src/pipelines/rust_app.rs

```diff
@@ -87,8 +96,8 @@ impl RustApp {
         let manifest = CargoMetadata::new(&path).await?;
         Ok(Self {
             id: None,
+            cargo_features: cfg.cargo_features.clone(),
```

Comment:
I really don't like this `clone` but don't see a good alternative without causing too many changes.